pkgs/development/libraries/mesa/default.nix

```diff
@@ -26,23 +26,27 @@
 
 with stdenv.lib;
 
+if ! elem stdenv.hostPlatform.system platforms.mesaPlatforms then
```

Comment:
This was removed intentionally. meta.platforms should handle this error

Reply:
Oh, sorry looks like I was comparing the changes with the master branch instead of staging during the rebase.